Banyan/src/sessions.jl

```diff
@@ -470,7 +476,7 @@ function run_session(;
                     directory = directory, dev_paths = dev_paths, force_sync = force_sync, force_pull = force_pull, force_install = force_install, 
                     estimate_available_memory = estimate_available_memory, nowait = false, email_when_ready = email_when_ready, for_running = true)
     catch
-        session_id = try
+        try
```

Comment:
I think we need to actually set `session_id` to either `get_session_id()` or `nothing.

So,

```
session_id = nothing
 try
            session_id = get_session_id()
        catch
            session_id = nothing
        end
```

Reply:
Resolved